src/main.rs

```diff
@@ -233,8 +286,18 @@ fn main() -> Result<()> {
                     println!("Bundle's CFPA would attempt to change an");
                     println!("RKTH_REVOKE bit from 1 to 0.  Only 0 to 1");
                     println!("transitions are allowed.");
-                    println!("bundle RKTH_REVOKE: {:02x}", cfpa.rkth_revoke);
-                    println!("device RKTH_REVOKE: {:02x}", current_cfpa.rkth_revoke);
+                    println!(
+                        "bundle RKTH_REVOKE: 0x{:02x} {}",
```

Comment:
You can convince `println!` to include `0x` for you automatically with the `#` format modifier; this also requires increasing count, because the `0x` is included::
```suggestion
                        "bundle RKTH_REVOKE: {:#04x} {}",
```

Reply:
Ah thank you for finding me the thing I could not remember.